client/scripts/models/IChainAdapter.ts

```diff
@@ -116,6 +115,16 @@ abstract class IChainAdapter<C extends Coin, A extends Account<C>> {
     console.log(`Stopping ${this.meta.chain.id}...`);
   }
 
+  public async loadModules(modules: ProposalModule<any, any, any>[]) {
```

Comment:
Want to add a comment saying that this can be called any number of times by the parent view?

I guess wrapping loadModules in a _.debounce() with trailing=true communicates this. However then we should also wrap the function so that invocations with different modules aren't accidentally lost. For example: https://github.com/lodash/lodash/issues/2403